cmd/kubeadm/app/util/config/common_test.go

```diff
@@ -202,207 +198,6 @@ func TestVerifyAPIServerBindAddress(t *testing.T) {
 	}
 }
 
-func TestMigrateOldConfigFromFile(t *testing.T) {
```

Comment:
i wonder what we should do about this unit test.
it is quite valuable to have it once we add v1beta4 or newer config.
it will allow us to unit test v1beta3 -> new migration from file.

so what we can do here is leave a TODO

```
// TODO: re-include TestMigrateOldConfigFromFile once a new API version is added after v1beta3.
// see <link-to-commit-foo> of how this unit test function
// looked before it was removed with the removal of v1beta2.
```

WDYT?

Reply:
+1 for retaining this test framework, it can be reusable in the future.

Reply:
+1, I hesitated to remove it when I did it.